pkg/s3utils/utils.go

```diff
@@ -126,6 +132,18 @@ func GetRegionFromURL(endpointURL url.URL) string {
 	if len(parts) > 1 {
 		return parts[1]
 	}
+	if IsAmazonFIPSUSEastWestEndpoint(endpointURL) {
```

Comment:
Unlike FIPS endpoiints in Govcloud, the normal FIPS endpoints are available in two locations - `us-east-1` and `us-west-1`. So, we need to extract out which one it is and return a value accordingly (as opposed to Gov cloud where we could just hardcode the `"us-gov-west-1"` location)